app/assets/javascripts/static_pages.js

```diff
@@ -0,0 +1,16 @@
+//= require medium-editor
+
+WebsiteOne.define('StaticPages', function() {
+    return {
+        init: function() {
+            var editor = new MediumEditor('.editable', {
+                placeholder: {
+                    /* This example includes the default options for placeholder,
+                        if nothing is passed this is what it used */
+                    text: 'Type your text',
+                    hideOnClick: true
+                }
+            }
+        };
+    }
+});
```

Comment:
Sure, although I'd recommend doing the simplest thing possible first just to check it works - I was just looking at the docs - why not just put the JavaScript tag in the view to start with?